_config.yml

```diff
@@ -36,27 +36,34 @@ ytkey: AIzaSyCiZ9uToWu9jb7BTx47NtzCvmGGXKXp8nI
 remote_theme: w3c/wai-website-theme
 
 collections:
-#  business-case:
-#    area: "Plan & Manage"
-#    name: "Business Case"
-#    shortname: "Business Case"
-#    icon: "book" # TODO
-#    repostitory: w3c/wai-bcase
-#    output: true
-#    acknowledgements: false
-#    permalink: /:collection/:path/
+  patterns:
+    title: Glossary
```

Comment:
This will [almost certainly] clash in the `wai-website`'s `_config.yml` as the `wai-coga-design-guide` includes a collection with the same name. I don't think `title` matters.

@shawna-slh @hidde we should figure out a naming convention for collections - how about prefix with the repo name? eg

- `wai-wcag-act-rules-patterns`
- `wai-coga-design-guide-patterns`

You can use whatever you like the the resource repo config but it's best to be consistent to avoid confusion

Reply:
I had assumed these names are local to the repository, but you know what they say about assumptions… :-) 

Agree with namespacing these, let's do that to be sure, great suggestion!

Reply:
Have updated the PR to namespace collection name.

Reply:
> I had assumed these names are local to the repository, but you know what they say about assumptions… :-) 

That's basically true, but wai-website sucks in all the repos and needs to declare all the collections in exactly the same way :(